tests/integration/features/fixtures/cluster_test_fixtures.go

```diff
@@ -44,6 +46,17 @@ func createOrUpdateSubscription(ctx context.Context, client client.Client, subsc
 	return err
 }
 
+func CreateOrUpdateDSCI(ctx context.Context,
+	client client.Client,
+	dsci *dsciv1.DSCInitialization) (*dsciv1.DSCInitialization, error) {
+	_, err := controllerutil.CreateOrUpdate(ctx, client, dsci, func() error {
+		return nil
+	})
+	dsci.APIVersion = dsciv1.GroupVersion.String()
+	dsci.Kind = gvk.DSCInitialization.Kind
+	return dsci, err
```

Comment:
Did not get why you need to return dsci if you modifying it inplace.

Reply:
Me neither :D I think we missed it in the first run with @cam-garrison.

Fixed in 5d25dc83